events/views.py

```diff
@@ -219,3 +225,22 @@ def stats(request):
 
     return render_to_response(
         'stats/index.html', data, context_instance=RequestContext(request))
+
+def events_category(request, id, slug):
+    """
+        Filter evets for category
+    """
+    try:
+        events = CategoryEvent.objects.filter(category_id=id)
```

Comment:
I'm confused, why do you need a ``CategoryEvent`` model? Having a field on events model with a foreignkey to category model should be enough.

Then here you would query with: 

``events = Events.objects.filter(category=id)``